notebooks/en/code_search.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>we could end on an enthusiastic tone, call for action build and give useful resources (maybe qdrant docs?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/151/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>